tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py

```diff
@@ -108,7 +107,7 @@ def test_ingest_consumer_reads_from_topic_and_calls_celery_task(
         consumer = get_ingest_consumer(
             max_batch_size=2,
             max_batch_time=5000,
-            group_id=random_group_id,
+            group_id="some-group-id",
```

Comment:
Why'd we change this btw? I'm not sure if it's the reason, but this test has been flakey since yesterday.

Reply:
https://sentry.io/organizations/sentry/issues/3616300100/

Reply:
Ironically I made the change in response to some review feedback suggesting not to use random group ids to avoid flaky tests, but I did so unfortunately quickly and thoughtlessly.

Reply:
https://github.com/getsentry/sentry/pull/39429

Reply:
I'm not actually sure whether this is the cause of flakiness, but we'll see if it fixes things afterwards